platform-core-module/src/main/java/com/softicar/platform/core/module/program/ProgramEnqueuer.java

```diff
@@ -34,7 +34,8 @@ private void enqueueExecution(AGUuid programUuid) {
 		AGProgram program = AGProgram.loadOrInsert(programUuid);
 		if (program.reloadForUpdate()) {
 			program//
-				.setQueuedAt(DayTime.now())
+				.getOrCreateProgramState()
+				.setQueuedAt(DayTime.now().truncateSeconds())
```

Comment:
We did not truncate seconds before - but we do now. Why so?

Reply:
Until commit 92c9622 the seconds were truncated.
With cron syntax the user can start programs only at the full minute. I think this is why the originator of the code decided to not display seconds to the user.

Reply:
Regardless of the history, I think that we should not truncate seconds here.
Otherwise, I am concerned that we might give ourselves a hard time, in concealing/faking the point in time at which a program was actually enqueued. Particularly, by discarding seconds, we're unable to tell whether there was a significant delay between the _intended_ (as defined by the schedule) and the _actual_ enqueuing of a program.
This kind of information might be relevant, not only to us as developers but also to an admin of a live instance.

Reply:
Ok, thanks for explanation!
I changed that.